torch/csrc/utils/python_arg_parser.h

```diff
@@ -345,6 +346,29 @@ inline c10::OptionalArray<int64_t> PythonArgs::intlistOptional(int i) {
   return intlist(i);
 }
 
+inline c10::OptionalArray<double> PythonArgs::doublelistOptional(int i) {
+  if (!args[i]) {
+    return {};
+  }
+  PyObject* arg = args[i];
+  // XXX no auto-broadcast
```

Comment:
`intlist` does automatic expansion from scalars to fixed-size lists.  We don't need that, right?

Reply:
I think it's fine to skip for now -- IMO we don't know if we need it or not yet.